packages/flutter_tools/lib/src/test/runner.dart

```diff
@@ -20,141 +20,179 @@ import 'flutter_web_platform.dart';
 import 'test_wrapper.dart';
 import 'watcher.dart';
 
-/// Runs tests using package:test and the Flutter engine.
-Future<int> runTests(
-  TestWrapper testWrapper,
-  List<String> testFiles, {
-  Directory workDir,
-  List<String> names = const <String>[],
-  List<String> plainNames = const <String>[],
-  bool enableObservatory = false,
-  bool startPaused = false,
-  bool disableServiceAuthCodes = false,
-  bool ipv6 = false,
-  bool machine = false,
-  String precompiledDillPath,
-  Map<String, String> precompiledDillFiles,
-  @required BuildMode buildMode,
-  bool trackWidgetCreation = false,
-  bool updateGoldens = false,
-  TestWatcher watcher,
-  @required int concurrency,
-  bool buildTestAssets = false,
-  FlutterProject flutterProject,
-  String icudtlPath,
-  Directory coverageDirectory,
-  bool web = false,
-  String randomSeed = '0',
-}) async {
-  // Configure package:test to use the Flutter engine for child processes.
-  final String shellPath = globals.artifacts.getArtifactPath(Artifact.flutterTester);
-  if (!globals.processManager.canRun(shellPath)) {
-    throwToolExit('Cannot execute Flutter tester at $shellPath');
-  }
+/// A class that abstracts launching the test process from the test runner.
```

Comment:
@jonahwilliams or @zanderso - should I leave a method here that redirects to `const _FlutterTestRunnerImpl().runTests` to avoid breaking internal customers?  I don't see evidence that we use this but not sure.

Reply:
FYI @chingjun, though I don't think this is used at all

Reply:
I don't think this is used internally. But whenever in doubt, you could always run the tests go/flutter-pr-test :)

Reply:
Ahh, good idea!